AWSCognitoAuth/AWSCognitoAuth.h

```diff
@@ -15,6 +15,9 @@
 
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
+#if __has_include(<AuthenticationServices/AuthenticationServices.h>)
+#import <AuthenticationServices/AuthenticationServices.h>
+#endif
```

Comment:
As discussed, do we need this check for an Apple framework?

Reply:
I manually tested this with iOS 14, 11 and 9.0 and it worked as expected.